src/plugins/MongoDB.ts

```diff
@@ -49,6 +56,7 @@ export default fp(async function MongoDB(server) {
   await mongoClient.connect();
 
   const db = {
+    Auth: await mongoClient.db('unciv').collection<Auth>('ErrorLogs'),
```

Comment:
The `ErrorLogs` looks like a copy mistake?

Reply:
yeah, fixed and almost everything rewritten